pkg/handlers/login.go

```diff
@@ -110,7 +110,29 @@ func (h *Handler) Login(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	loginResponse.Token = fmt.Sprintf("Bearer %s", signedJWT)
+	responseToken := fmt.Sprintf("Bearer %s", signedJWT)
```

Comment:
`Bearer` might not be necessary here since we're just storing it in a cookie, but I see that it does provide value in the `Parse(...)` function where we determine if it's a session token or KOTS cli token.  Probably not worth changing, but just a random thought...

Reply:
Right, this way the parse function does not need to change.